torchtune/training/checkpointing/_utils.py

```diff
@@ -330,6 +329,7 @@ def copy_files(
     output_dir: Union[str, Path],
     *,
     ignore_suffixes: Optional[List[str]] = None,
+    max_file_size_mb: int = 100,
```

Comment:
Is there actually a way for someone to modify these values from the CLI/config? Seems like no, right?

Reply:
no, but i dont see it being an issue. It shouldnt happen, and If users really want to come a 100MB file to every epoch, they can do manually.